feat: prevent idling by playing silent audio

a-higuchi opened the pull request:
# Description
## Current problem
There was a problem that the beginning of the audio was not played in the STATE_WAITING_CALL_PERMISSION state.  
On some devices, after stopping audio playback for a period of time, the audio device automatically goes idle and does not play the beginning of the audio when it returns from idle.  

## Countermeasures
Change the stopped portion of the BGM to silent playback. 
This change prevents audio devices from becoming idle while the system is running. 

## Confirmation details
Execute the following command.
```
$ ros2 topic pub /autoware_state_machine/state autoware_state_machine_msgs/msg/StateMachine '{service_layer_state: 201, control_layer_state: 1}' --qos-durability transient_local --qos-reliability reliable --once
$ ros2 topic pub /autoware_state_machine/state autoware_state_machine_msgs/msg/StateMachine '{service_layer_state: 250, control_layer_state: 1}' --qos-durability transient_local --qos-reliability reliable --once
```
### Before measures:  
call.wav is played in a loop, but the beginning of the audio playback after the second time is cut off.

### After measures:  
Loop playback of call.wav works correctly.

## Review Procedure

 * Make sure the description is valid.
 * Make sure that what is described in the description matches the implementation.

a-higuchi committed:
fix/audio playback cut off

yn-mrse commented:
@a-higuchi 
Thank you for your contribution!
Please describe the test method that was specifically carried out.

yn-mrse reviewed (changes requested):
@a-higuchi 
Could you please add the purpose and function of this silent audio playback to the README?
If this is not documented, the design becomes difficult to maintain.

a-higuchi committed:
fix(readme): added about silent playback of BGM

a-higuchi commented:
> @a-higuchi Could you please add the purpose and function of this silent audio playback to the README? If this is not documented, the design becomes difficult to maintain.

Added silent playback of BGM to Readme.

a-higuchi commented:
> > @a-higuchi Could you please add the purpose and function of this silent audio playback to the README? If this is not documented, the design becomes difficult to maintain.

Added silent playback of BGM to Readme.

> @a-higuchi Thank you for your contribution! Please describe the test method that was specifically carried out.

Added specific test method.

a-higuchi committed:
fix(readme): correction of README contents

a-higuchi committed:
fix(readme): change README configuration
